Add cookieStoreId to contentScripts.register (#11268)
mdn/content: files/en-us/mozilla/firefox/releases/97/index.md

## Changes for add-on developers

- `cookieStoreId` in {{WebExtAPIRef("tabs.query")}} supports an array of strings. This enables queries to match tabs against more than one cookie store ID ({{bug(1730931)}}).

> - `cookieStoreId` added to {{WebExtAPIRef("contentScripts.register")}}. This enables script to be registered to the tabs for a contextual identity ({{bug(1470651)}}).

Reviewer: "enabled script"? I think that "script" would need to be plural ("scripts").
I'm not sure what you mean by "to the tabs for a contextual identity". It would be "to container tabs" or "limited to specific contextual identities", etc.

I'm not fully satisfied with "container tabs" or "contextual identities" since "cookieStoreId" encompasses more than that. Perhaps "container tabs or (private) browsing modes" would be the most complete (it's a mouthful though).

Here is an example of a message that matches closer with what I'm thinking:

```suggestion
- `cookieStoreId` added to {{WebExtAPIRef("contentScripts.register")}}. This allows extensions to register container-specific content scripts ({{bug(1470651)}}).
```
